.changeset/good-walls-brush.md

```diff
@@ -0,0 +1,18 @@
+---
+"synckit": minor
+---
+
+feat: add support for `--experimental-strip-types`
```

Comment:
I almost forgot request changing `README` doc, you use the changeset content as reference, or even help to improve the sentences because I'm not a native English speaker.